### ISSUE TITLE
Could 'gulp build' css or js in one file?

### ISSUE THREAD

**Issue by sinoon** [Semantic-Org/Semantic-UI#3776]
After use `gulp build`,will create a lot of file like:
```
accordion.min.css
accordion.min.js
ad.min.css
api.js
api.min.js
breadcrumb.css
breadcrumb.min.css
button.css
button.min.css
```
but,I am using `webpack`,so,I have to import them file by file.
i think is unnecessary,because when  during run `gulp install ` , I already select what component I need

Is there some way could build all components in one file?
like:
```
semantic-ui.min.js
semantic-ui.min.css
```

**Comment by sinoon** [comment]
I find!
Just run `gulp 'package compressed css'`

**Comment by sinoon** [comment]
close #3776

**Comment by sinoon** [comment]
Closed #3776
